WooCommerce/Classes/ViewRelated/Orders/Refund/RefundSubmissionUseCase.swift

```diff
@@ -107,6 +107,11 @@ final class RefundSubmissionUseCase: NSObject, RefundSubmissionProtocol {
         self.stores = stores
         self.storageManager = storageManager
         self.analytics = analytics
+
+        // Instantiates the alerts coordinator.
+        let alerts = OrderDetailsPaymentAlerts(transactionType: .refund,
+                                               presentingController: rootViewController)
+        self.alerts = alerts
```

Comment:
there might be a merge conflict from https://github.com/woocommerce/woocommerce-ios/pull/6639 which DIs the `alerts` so this isn't needed anymore 🙂